src/config.js

```diff
@@ -38,7 +38,7 @@ const PushNotificationsConfigs = {
                   id: data.post_id,
                 });
               } 
-             
+			
             }
           } catch (error) {
             console.log(error);

```

Comment:
remove the habit of making console on prod application. logs should be enabled for debugging app only